src/Implement/UnitTestAuthenticator.cpp

```diff
@@ -0,0 +1,10 @@
+#include "Authenticator.h"
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
+
+TEST(Authenticator_TEST, SimpleTest) {
+  std::string test_name = "test", test_pwd = "test";
+  Authenticator *test_target = Authenticator::GetInstance();
+  EXPECT_EQ(true, test_target->CheckAccount(test_name, test_pwd));
+  EXPECT_EQ("", test_name);
```

Comment:
Should this line be:
EXPECT_EQ("", test_target->SearchAccount(test_name));
Otherwise, the test wouldn't pass.

Reply:
Done

Reply:
This is a reasonable code review.